feat(changelog): generative-apis-added-new-model-support-devstral-small 2025-07-29 (#5343)
scaleway/docs-content: changelog/july2025/2025-07-29-generative-apis-added-new-model-support-devstral-small.mdx

---

> title: New model support Devstral Small

Reviewer: ```suggestion
title: New supported model - Devstral Small
```

status: added
date: 2025-07-29
category: ai-data
product: generative-apis
---

> [Devstral small 2505](https://www.scaleway.com/en/docs/generative-apis/reference-content/supported-models/#chat-models) is now available in Preview in Generative APIs.

Reviewer: ```suggestion
[Devstral Small 2505](https://www.scaleway.com/en/docs/generative-apis/reference-content/supported-models/#chat-models) is now available in preview for Generative APIs.
```

> Devstral small is a fine-tuned version of mistral small 3.1 optimized to perform agentic coding tasks, that can be plugged [into your favorite IDE](https://www.scaleway.com/en/docs/generative-apis/reference-content/integrating-generative-apis-with-popular-tools/).

Reviewer: ```suggestion
Devstral Small is a fine-tuned version of Mistral Small 3.1, optimized to perform agentic coding tasks. It can be plugged [into your favorite IDE](/generative-apis/reference-content/integrating-generative-apis-with-popular-tools/).
```
